docs/roadmap/planning.rst

```diff
@@ -0,0 +1,37 @@
+:title: Planning Process
+:description: The open source planning process for the Deis project used to define the roadmap and its execution.
+
+.. _planning:
+
+Planning Process
+================
+Deis features a lightweight process that emphasizes openness and ensures every community member can be an integral part of planning for the future.
+
+The Role of Maintainers
+-----------------------
+Maintainers lead the project. Their duties include proposing the Roadmap, reviewing and integrating contributions and maintaining the vision of the project.
+
+Open Roadmap
+------------
+The :ref:`roadmap` is a community document. While Maintainers propose the Roadmap, it gets discussed and refined in Sprint Meetings.
+
+Contributing to the Roadmap
+---------------------------
+Proposals and issues can be opened by anyone. Every member of the :ref:`community` is welcome to participate in the discussion by providing feedback and/or offering counter-proposals.
+
+Release Milestones
+------------------
+The Roadmap gets delivered progressively via the :ref:`release_schedule`.  Releases are defined during Sprint Meetings and managed using GitHub Milestones which track specific deliverables and work-in-progress.
+
+Sprint Meetings
```

Comment:
May want to call these "Release Planning Meeting" or something like that. "Sprint" can give folks a rash. :)